pytest_fixtures.py

```diff
@@ -0,0 +1,90 @@
+import pytest
+
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        '--test_db_url',
```

Comment:
@juliotrigo take note.

@kooba did you intentionally drop the `db-dialect` and `db-driver` options? @juliotrigo, what was your reasoning for having them separately?